contracts/HenkakuBadge.sol

```diff
@@ -85,7 +86,15 @@ contract HenkakuBadge is ERC1155, Ownable {
     }
 
     // of can be token id or address
```

Comment:
Not from this PR but

> of can be token id or address

I guess we need both "of" and "token id" separately, and they have different types.